enhancements/ingress/set_eip_nlb_ingress.md

```diff
@@ -0,0 +1,1319 @@
+    controller: true
+    kind: Deployment
+    name: router-default
+    uid: 6d547566-82f0-4a35-96bf-afd18c1fdc3e
+  resourceVersion: "197756"
+  uid: 84915856-2015-4d59-92dc-decc06c5779e
+spec:
+  allocateLoadBalancerNodePorts: true
+  clusterIP: 172.30.128.122
+  clusterIPs:
+  - 172.30.128.122
+  externalTrafficPolicy: Local
+  healthCheckNodePort: 32623
+  internalTrafficPolicy: Cluster
+  ipFamilies:
+  - IPv4
+  ipFamilyPolicy: SingleStack
+  ports:
+  - name: http
+    nodePort: 31065
+    port: 80
+    protocol: TCP
+    targetPort: http
+  - name: https
+    nodePort: 31778
+    port: 443
+    protocol: TCP
+    targetPort: https
+  selector:
+    ingresscontroller.operator.openshift.io/deployment-ingresscontroller: default
+  sessionAffinity: None
+  type: LoadBalancer
+status:
+  loadBalancer:
+    ingress:
+    - hostname: a8491585620154d5992dcdecc06c5779-7938285b62f7bebe.elb.us-east-1.amazonaws.com
+```
+
+2. What happens to the subnets added in the installer config?
+
+Addition of subnets is a different EP. However, the eips should match the number of public subnets which is a user responsibility.
+More details on how to check the number of subnets are mentioned [here](#Set EIP using an installer)
+
+3. What happens when someone updates the eipAllocations for the load balancer service of an Ingress Controller ?
+
+It is not supported to manually edit the eipAllocations for the load balancer service of an Ingress Controller.
+However, if someone does that then they will find the error in status as follows in the IngressController:
+```yaml
+  - lastTransitionTime: "2024-04-17T03:09:32Z"
+    message: |-
+      The service-controller component is reporting SyncLoadBalancerFailed events like: Error syncing load balancer: failed to ensure load balancer: error creating load balancer: "ResourceInUse: The allocation IDs are not available for use\n\tstatus code: 400, request id: 58136c42-e1bd-4c1a-ac38-7078da2b1d95"
+      The kube-controller-manager logs may contain more details.
+    reason: SyncLoadBalancerFailed
+    status: "False"
+    type: LoadBalancerReady
+  - lastTransitionTime: "2024-04-17T02:54:25Z"
+    message: LoadBalancer is not progressing
+    reason: LoadBalancerNotProgressing
+    status: "False"
+    type: LoadBalancerProgressing
+...
+...
+  - lastTransitionTime: "2024-04-17T03:09:32Z"
+    message: |-
+      One or more status conditions indicate unavailable: LoadBalancerReady=False (SyncLoadBalancerFailed: The service-controller component is reporting SyncLoadBalancerFailed events like: Error syncing load balancer: failed to ensure load balancer: error creating load balancer: "ResourceInUse: The allocation IDs are not available for use\n\tstatus code: 400, request id: 58136c42-e1bd-4c1a-ac38-7078da2b1d95"
+      The kube-controller-manager logs may contain more details.)
+    reason: IngressControllerUnavailable
+    status: "False"
+    type: Available
+
+```
+
+- Are there any pre-requisites for this EP ?
```

Comment:
These are numbered questions above, please continue the numbering here and below.  But also, these are not Open Questions if they have answers.

Reply:
changed